dali/pipeline/operators/reader/loader/lmdb.h

```diff
@@ -24,71 +24,153 @@
 
 namespace dali {
 
-#define CHECK_LMDB(status, filename) \
+#define CHECK_LMDB(status) \
   do { \
     DALI_ENFORCE(status == MDB_SUCCESS, "LMDB Error: " + string(mdb_strerror(status)) + \
-                                        ", with file: " + filename); \
+                                        ", with file: " + db_path_); \
```

Comment:
I don't think it is a good idea to use a hardcoded `db_path_` here because it makes this macro unusable if that member variable does not exist. Can you revert back to receiving the `filename` as an argument?